### DISCUSSION TITLE
without .css  after update to 8.3.1 dissapoint file .css theme

### DISCUSSION THREAD

**Discussion by piotrek2555** [LMS-Community/slimserver#929] [General]
After 8.3.1 my rpi 4b, www lms looks like:

[url=https://ibb.co/C13XtCJ][img]https://i.ibb.co/ngNV0d3/lms-without-css.png[/img][/url]

squeezelite-x on windows 10 display theme .css material skin

**Comment by mherger** [comment]
Please fix the link in your posting and describe what exactly you're doing, and what is happening.

**Comment by piotrek2555** [comment]
I fixed it,  download material skin and install
